.github/labeler.yml

```
@@ -0,0 +1,21 @@
+documentation:
+- doc/**/*
+- examples/**/*
+- examples_flask/**/*
+- ./*.md
+maintenance:
+- .github/**/*
+- codecov.yml
+- .flake8
+- .coveragerc
+- .codecimate.yml
+- Makefile
+- ignore_words.txt
+- mypy.ini
+- pyproject.toml
+dependencies:
+- requirements*
+- environment.yml
+docker:
+- docker/**/*
+- .dockerignore
```

Just removed non-existing files in pymapdl project:

```suggestion
documentation:
- doc/**/*
- examples/**/*
maintenance:
- .github/**/*
- codecov.yml
- .flake8
- .coveragerc
- ignore_words.txt
- pyproject.toml
dependencies:
- requirements*
docker:
- docker/**/*
```